src/jsMain/kotlin/dev/bitspittle/firebase/auth/Bindings.ActionInfo.kt

```diff
@@ -0,0 +1,27 @@
+package dev.bitspittle.firebase.auth
+
+import dev.bitspittle.firebase.externals.auth.ActionCodeSettings
+import dev.bitspittle.firebase.util.jsonWithoutNullValues
+
+class ActionCodeSettings(
+    android: Android,
+    dynamicLinkDomain: String?,
+    override val handleCodeInApp: Boolean,
+    iOS: Ios,
+    override val url: String
+): ActionCodeSettings {
+    class Android(val installApp: Boolean?, val minimumVersion: String?, val packageName: String)
+    class Ios(val bundleId: String)
+
+    override val android: dynamic = jsonWithoutNullValues(
+        "installApp" to android.installApp,
+        "minimumVersion" to android.minimumVersion,
+        "packageName" to android.packageName,
+    )
+
+    override val dynamicLinkDomain: String? = dynamicLinkDomain ?: undefined // null != undefined
+
+    override val iOS: dynamic = jsonWithoutNullValues(
```

Comment:
Same comment as "undefined" above -- but here, "dynamic". Let's never expose that to users. (This should probably be `val iOS: Ios` or something like that)